iroha-cpp/services/Query.cpp

```diff
@@ -0,0 +1,163 @@
+#include "GrpcClient.hpp"
+#include "cryptography/ed25519_sha3_impl/internal/ed25519_impl.hpp"
+#include "GrpcResponseHandler.hpp"
+#include "services/Query.hpp"
+#include "model/converters/pb_common.hpp"
+
+
+namespace IROHA_CPP
+{
+
+Query::Query(const std::string& account_name, uint64_t query_counter, const iroha::keypair_t& keypair)
+    : creator_(account_name),
+      counter_(query_counter),
+      keypair_(keypair)
+{}
+
+void Query::populateQueryMetaData(iroha::protocol::Query& pb_query)
+{
+    auto* meta = pb_query.mutable_payload()->mutable_meta();
+    meta->set_created_time(local_time_);
+    meta->set_creator_account_id(creator_);
+    meta->set_query_counter(counter_);
+}
+
+Query& Query::getAccount(const std::string& account_id)
+{
+    iroha::protocol::Query pb_query;
+    auto pl = pb_query.mutable_payload();
+    populateQueryMetaData(pb_query);
+
+    auto pb_query_mut = pl->mutable_get_account();
+    pb_query_mut->set_account_id(account_id);
+
+    queryProto = pb_query;
+    return *this;
+}
+
+Query& Query::getAccountAssets(const std::string& account_id)
+{
+    iroha::protocol::Query pb_query;
+    populateQueryMetaData(pb_query);
```

Comment:
isn't better to do its like that (function name probably should be different after the change):
```suggestion
    iroha::protocol::Query pb_query = populateQueryMetaData();
```
or even:
```suggestion
    auto pb_query = populateQueryMetaData();
```

Reply:
Outdated comment.